sfdoc/publish/salesforce.py

```diff
@@ -152,7 +152,18 @@ def create_article(self, html):
         data = html.create_article_data()
         data[settings.SALESFORCE_DOCSET_RELATION_FIELD] = self.sf_docset['Id']
         assert data[settings.SALESFORCE_DOCSET_RELATION_FIELD]
-        result = kav_api.create(data=data)
+        try:
+            result = kav_api.create(data=data)
+        except SimpleSalesforceExceptions.SalesforceMalformedRequest:
+            sf_api_logger.error(f"Error publishing {data['UrlName']}")
+            art = Article.objects.filter(url_name=data['UrlName']).last()
+            old, new = art.docset_id, self.sf_docset['Id']
+            if old!=new:
```

Comment:
Doesn't need to block this, but can we run black on sfdoc and set it up to run in a pre-commit hook?